includes/api/class-edd-api.php

```diff
@@ -1492,6 +1501,86 @@ public function get_discounts( $discount = null ) {
 	}
 
 	/**
+	 * Process Get Downloads API Request
+	 *
+	 * @access public
+	 * @since 2.5
+	 * @author Daniel J Griffiths
+	 * @global object $wpdb Used to query the database using the WordPress
+	 *   database API
+	 * @return array $downloads Multidimensional array of the downloads
+	 */
+	public function get_downloads( $customer ) {
+		$downloads  = array();
+		$errors     = array();
+		$meta_query = array();
+
+		global $wpdb, $edd_logs;
+
+		$paged    = $this->get_paged();
+		$per_page = $this->per_page();
+		$offset   = $per_page * ( $paged - 1 );
+		$count    = 0;
+
+		if( $customer ) {
+			if( is_numeric( $customer ) ) {
+				$meta_query[] = array(
+					'key'    => '_edd_log_user_id',
+					'value'  => $customer
```

Comment:
You are using `$customer` but the meta query is for `user`. We need to stop making this confusion. If it's `customer`, use customer ids, if it's `user`, use user ids. But this is just itching to be confused and giving wrong information.